src/bootstrap/config.rs

```diff
@@ -260,6 +261,7 @@ impl Target {
 #[derive(Deserialize, Default)]
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 struct TomlConfig {
+    version: Option<usize>,
```

Comment:
We should require this, I think? Is there a reason not to?

Reply:
If it's mandatory, that breaks everyone with `config.toml` currently (a hard break, the TOML won't parse). If we make it optional, they just get a warning to look at the changelog.